src/mscorlib/shared/System/Guid.Windows.cs

```diff
@@ -15,7 +15,7 @@ public static Guid NewGuid()
             // how extensively it checks for known values.
 
             Guid g;
-            int hr = Interop.mincore.CoCreateGuid(out g);
+            int hr = Microsoft.Win32.Win32Native.CoCreateGuid(out g);
```

Comment:
This will break CoreRT. Could you please copy over https://github.com/dotnet/corert/blob/master/src/Common/src/Interop/Windows/mincore/Interop.CoCreateGuid.cs, rename it Ole32, and delete `Microsoft.Win32.Win32Native.CoCreateGuid` ?